### PR TITLE
Update migrate-az-9.0.0-servicebus.md

### PR DESCRIPTION
**Pull request by Saglodha**
Adding additional breaking changes for cmdlets.

# PR Summary

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md [hunk 1]
```diff
@@ -187,13 +222,15 @@ Use Set-AzServiceBusNetworkRuleSet to add/remove multiple IP/ virtual network ru
 - Parameter `-EnableBatchedOperations` would be renamed to `-EnableBatchedOperation`.
 - Input type of parameters `-DefaultMessageTimeToLive`, `-AutoDeleteOnIdle`, `-DuplicateDetectionHistoryTimeWindow` has been changed from System.String to System.Timespan. Hence, ISO 8601 format for timespan can NO longer be fed as input to these parameters. Please use New-TimeSpan cmdlet object to construct Timespan variables. Please refer [New-TimeSpan](/powershell/module/microsoft.powershell.utility/new-timespan) to know more about New-TimeSpan.
 - Parameter `-ResourceGroupName` would no longer support alias `-ResourceGroup`.
+- Parameter 'Namespace' has been renamed to 'NamespaceName'.
```
Comment (damodaravadhani) [inline docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md:225]: Replace single quote (') with (`) for parameter names so they are displayed with a grey background everywhere. @Saglodha

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
Looks good other than the changes @damodaravadhani requested: _Replace single quote (') with (`) for parameter names so they are displayed with a grey background everywhere._

File: docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md [hunk 2]
```diff
@@ -187,13 +222,15 @@ Use Set-AzServiceBusNetworkRuleSet to add/remove multiple IP/ virtual network ru
 - Parameter `-EnableBatchedOperations` would be renamed to `-EnableBatchedOperation`.
```
Comment (damodaravadhani) [inline docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md:222]: Let's remove this also. The rename isn't happening. @Saglodha

**Review by damodaravadhani — CHANGES_REQUESTED** [review]

File: docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md [hunk 3]
```diff
@@ -142,25 +167,33 @@ Use Set-AzServiceBusNetworkRuleSet to add/remove multiple IP/ virtual network ru
 - `-SizeInBytes` and `-MessageCount` are readonly parameters and are getting removed.
 - Parameter `-EnableBatchedOperations` is renamed to `-EnableBatchedOperation`.
 - Input type of parameters `-DefaultMessageTimeToLive`, `-AutoDeleteOnIdle`, `-LockDuration`, `-DuplicateDetectionHistoryTimeWindow` has been changed from System.String to System.Timespan. Hence, ISO 8601 format for timespan can NO longer be fed as input to these parameters. Please use New-TimeSpan cmdlet object to construct Timespan variables. Please refer [New-TimeSpan](/powershell/module/microsoft.powershell.utility/new-timespan) to know more about New-TimeSpan.
```
Comment (damodaravadhani) [inline docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md:169]: Let's remove this @Saglodha

Comment (damodaravadhani) [reply docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md:169]: The rename to to EnableBatchedOperation is not happening

**Comment by damodaravadhani** [comment]
Thanks @Saglodha for the fixes. Looks good to me !

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Pull request merged**